Makefile

```diff
@@ -185,6 +185,8 @@ node-problem-detector:
 projects-secrets:
 	# GITHUB_DEPLOYMENTCONFIG_TRIGGER is used to route github webhook deliveries
 	oc create secret generic github-deploymentconfig-trigger --from-literal=WebHookSecretKey="${GITHUB_DEPLOYMENTCONFIG_TRIGGER}"
+	# IMAGE_REGISTRY_PUBLISHER_BOT_GITHUB_TOKEN is used to push changes from github.com/openshift/image-registry/vendor to our forked repositories.
+	oc create secret generic -n image-registry-publishing-bot github-token --from-literal=token=$${IMAGE_REGISTRY_PUBLISHER_BOT_GITHUB_TOKEN?} --dry-run -o yaml | oc apply -f -
```

Comment:
I use the syntax ${variable?} (with escaped $) to get an error from the shell when the variable is not set.